packages/otetris/otetris.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "lord@crocodile.org"
+authors: "Vadim Zaliva"
+homepage: "https://github.com/vzaliva/otetris"
+bug-reports: "https://github.com/vzaliva/otetris/issues"
+license: "MIT"
+dev-repo: "https://github.com/vzaliva/otetris.git"
+build: ["jbuilder" "build"]
+install: ["jbuilder" "install"]
+remove: [
+  ["rm" "%{bin}%/otetris"]
+  ["rm" "%{bin}%/ttetris"]
+]
+depends: [
+  "batteries"
```

Comment:
can you add a constraint to `"jbuilder" {build}`? This will help to give a proper error when one wants to install the package on OCaml < 4.02.